src/app/services/actions/permissions-action-creator.spec.ts

```diff
@@ -198,18 +198,6 @@ describe('Permissions action creators', () => {
     expect(urlScopesAction).toEqual(expectedUrlScopesAction)
   });
 
```

Comment:
What's the corresponding expected test with the new changes? Feels like we have introduced new functionality without test coverage.

Reply:
Actually, there was no need to remove this test. I will restore it.